src/collection/dimensions.js

```diff
@@ -266,6 +266,33 @@ fn = elesfn = ({
         };
       }
 
+      function computePaddingValues( width, height, paddingObject, relativeTo ) {
+        // Assuming percentage is number from 0 to 1
```

Comment:
Ok.  Maybe the comment is fine,  I just didn't understand how the pfValue worked in this case.